sdk/core/core/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0-dev.1 (Unreleased)
+## 1.0.0-preview.1 (Unreleased)
 
 * Testing. Validating automation.
```

Comment:
Delete the text under the header so we don't confuse people when it's time for release.